WeakAuras/WeakAuras.lua

```diff
@@ -1445,6 +1451,7 @@ loadFrame:RegisterEvent("UNIT_EXITED_VEHICLE");
 loadFrame:RegisterEvent("SPELLS_CHANGED");
 loadFrame:RegisterEvent("GROUP_JOINED");
 loadFrame:RegisterEvent("GROUP_LEFT");
+loadFrame:RegisterEvent("UNIT_AURA");
```

Comment:
I thought about this as well but when I actually tested it the performance impact was very negligible. But alright I'll try to find a better way